pack/tfa/tof_encounter.json

```diff
@@ -1,164 +1,252 @@
         "clues": 2,
         "code": "04120",
         "double_sided": true,
         "encounter_code": "threads_of_fate",
+        "encounter_position": 8,
         "faction_code": "mythos",
-        "flavor": "One of the museum security guards, a man named Drew Walker, recognizes you from your prior work with Alejandro and Dr. Horowitz. “Look, I ain’t supposed to be letting anybody into the exhibit…But the coppers don’t know from nothing when it comes to this hokum.” He steps aside and lets you pass. “If anybody asks, you slipped in while I was using the john, all right? Don’t take any wooden nickels.”",
+        "flavor": "One of the museum security guards, a man named Drew Walker, recognizes you from your prior work with Alejandro and Dr. Horowitz. \"Look, I ain’t supposed to be letting anybody into the exhibit... But the coppers don’t know from nothing when it comes to this hokum.\" He steps aside and lets you pass. \"If anybody asks, you slipped in while I was using the john, all right? Don’t take any wooden nickels.\"",
         "illustrator": "Joshua Cairós",
         "name": "At the Exhibit",
         "pack_code": "tof",
         "position": 120,
         "quantity": 1,
+        "stage": 2,
         "text": "<b>Objective</b> - Only investigators in the Eztli Exhibit can spend the requisite number of clues, as a group, to advance.",
         "type_code": "act"
     },
     {
-        "back_text": "<em class='bbc'>Harlan finally snaps out of his daze, returning to his normal self. He tells you that he was accosted by several figures in brown robes, who asked him where he'd hidden the relic. They used some kind of hypnotism to pry the knowledge from him by force. He had no choice but to tell them the truth&mdash;that he'd brought the relic to the Curiositie Shoppe and gave it to one of his associates. If you make haste, perhaps you can reach the relic before the brown-robed figures do!<br/></em>Attach the set-aside Relic of Ages to the Curiositie Shoppe.<br/>If there are 2 or fewer act decks in play, search the encounter deck for 1 Acolyte (2 Acolytes instead if there are 3 or 4 investigators in the game). Spawn each searched-for Acolyte in the Curiositie Shoppe.<br/>Advance to Act 3a-\"Find the Relic.\"",
+        "back_flavor": "Harlan finally snaps out of his daze, returning to his normal self. He tells you that he was accosted by several figures in brown robes, who asked him where he'd hidden the relic. They used some kind of hypnotism to pry the knowledge from him by force. He had no choice but to tell them the truth&mdash;that he'd brought the relic to the Curiositie Shoppe and gave it to one of his associates. If you make haste, perhaps you can reach the relic before the brown-robed figures do!",
```

Comment:
There's &mdash; in this line due to be deleted and replace with space-dash-space.